test/torchaudio_unittest/functional/librosa_compatibility_test.py

```diff
@@ -109,6 +109,25 @@ def test_amplitude_to_DB(self):
 
         self.assertEqual(ta_out, lr_out, atol=5e-5, rtol=1e-5)
 
+    def test_resample(self):
+        input_path = common_utils.get_asset_path('sinewave.wav')
+        waveform, sample_rate = common_utils.load_wav(input_path)
+
+        upsample_rate = sample_rate * 2
+        downsample_rate = sample_rate // 2
+
+        ta_upsampled = F.resample(waveform, sample_rate, upsample_rate)
+        lr_upsampled = librosa.resample(waveform.squeeze(0).numpy(), sample_rate, upsample_rate)
+        lr_upsampled = torch.from_numpy(lr_upsampled).unsqueeze(0)
+
+        self.assertEqual(ta_upsampled, lr_upsampled, atol=1e-4, rtol=1e-5)
```

Comment:
@vincentqb @cpuhrsch what's an acceptable margin of error for this comparison? using these values gives an assertion error ` Tensors failed to compare as equal!With rtol=1e-05 and atol=0.0001, found 28142 element(s) (out of 128000) whose difference(s) exceeded the margin of error (including 0 nan comparisons). The greatest difference was 0.006743520498275757 (0.23187145590782166 vs. 0.2386149764060974), which occurred at index (0, 127992).`

Reply:
You can look at the [kaldi compliance](https://github.com/pytorch/audio/blob/master/test/torchaudio_unittest/compliance_kaldi_test.py#L198) as guidance. As you can see, the tolerances are fairly high there too.

Since we are not changing the algorithm as part of this PR, these new tests are informational, and we take the threshold that makes them pass (after we make sure that we match the correct parameters of course). The threshold therefore becomes new signal/information that we learned from this exercise.

IMHO, based on the numbers you gave, assuming we use the correct corresponding parameters for each resampling functions, I'd say `atol=1e-2` which is also used [here](https://github.com/pytorch/audio/blob/master/test/torchaudio_unittest/compliance_kaldi_test.py#L184).

Reply:
I had only tried the librosa default values for this test, but yup, I can look into the other parameters